tests/HypothesisClient/Exception/ApiExceptionTest.php

```diff
@@ -34,7 +26,7 @@ public function it_requires_a_message()
      */
     public function it_has_an_error_code_of_zero()
     {
-        $e = $this->createMock(ApiException::class);
+        $e = new ApiException('foo');
         $this->assertEquals(0, $e->getCode());
```

Comment:
Assertion on a mock. 🙃

Reply:
If this was `assertSame` it would have correctly failed due to the mocked method returning `null`?

Reply:
Apparently not. 🤔

`Exception::getCode()` is `final`, so must be a default.

Reply:
But yes, should be `assertSame` anyway.